drivers/flash/soc_flash_nrf.c

```diff
@@ -23,15 +23,30 @@
 #include "controller/ticker/ticker.h"
 #include "controller/include/ll.h"
 
+#if defined(CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE)
+
+/* The timeout is multiplied by 1.5 because switching tasks may take
+ * significant portion of time.
+ */
+#define FLASH_TIMEOUT_MS ((FLASH_PAGE_ERASE_MAX_TIME_US)\
+			* (FLASH_PAGE_MAX_CNT) / 1000 * 15 / 10)
+
+#else
+
+#define FLASH_TIMEOUT_MS ((FLASH_PAGE_ERASE_MAX_TIME_US)\
+			* (FLASH_PAGE_MAX_CNT) / 1000)
+
+#endif /* CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE */
+
 #define FLASH_SLOT_ERASE     FLASH_PAGE_ERASE_MAX_TIME_US
 #define FLASH_INTERVAL_ERASE FLASH_SLOT_ERASE
+
+#endif /* CONFIG_SOC_FLASH_NRF_RADIO_SYNC */
```

Comment:
It seems that this line has been unnecessarily moved up. You don't need the three below definitions when `CONFIG_SOC_FLASH_NRF_RADIO_SYNC` is not enabled.

Reply:
OK